variables.tf

```diff
@@ -33,6 +33,26 @@ variable "options_indices_query_bool_max_clause_count" {
   default     = "1024"
 }
 
+variable "cognito_enabled" {
+  description = "Bool(optional, false): Whether to enable Cognito for authentication in Kibana"
+  default     = false
+}
+
+variable "cognito_user_pool_id" {
+  description = "String(required) ID of the Cognito User Pool to use"
```

Comment:
Only a small thing, I would clarify that it's required if `cognito_enabled` is `true`, same for the other 2 vars

Reply:
indeed was doubting. Changed now